api/_globals/console.json

```
@@ -1449,6 +1449,289 @@
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "nodejs": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": false,
+              "deprecated": false
+            }
+          }
+        },
+        "end_paramter": {
+          "__compat": {
+            "description": "`end` parameter",
+            "support": {
+              "chrome": {
+                "version_added": "134"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "nodejs": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": true,
+              "standard_track": false,
+              "deprecated": false
+            }
+          }
+        },
+        "label_parameter": {
+          "__compat": {
+            "description": "`label` parameter",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/console/timeStamp_static#label",
+            "support": {
+              "bun": {
+                "version_added": "1.0.0"
+              },
+              "chrome": {
+                "version_added": "15"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": "1.29"
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": [
+                {
+                  "version_added": "98",
+                  "impl_url": "https://bugzil.la/1387528",
+                  "partial_implementation": true,
+                  "notes": "The method can be called, but the marker does not show up in the performance profile."
+                },
+                {
+                  "version_added": "39",
+                  "version_removed": "97"
+                }
+              ],
```

Effectively the `label` parameter is not supported, see: https://searchfox.org/firefox-main/rev/e613f4df351a21871cfeadf7d5b4043ffad157b1/dom/webidl/Console.webidl#68

```suggestion
              "firefox": {
                "version_added": false
              },
```